packages/web/graphql/fragments.graphql

```diff
@@ -9,6 +9,9 @@ fragment UserFragment on User {
   city
   country
   emailAddressVerified
+  configuration {
+    ...UserConfigurationFragment
+  }
```

Comment:
We user this fragment in several places, including profile pages. We probably don't want to to include the general case. If we don't already have one, maybe a "SettingsPageUserFragment" would make sense? Or maybe having this only be a part of the top level query instead of including it in any fragment?

Reply:
Yep good call too, moved to the settings data query thing!